src/data/familiars.txt

```diff
@@ -317,6 +317,6 @@
 285	Arachnelf	arachnelf.gif	none	festive egg sac	Site Alpha ID badge	3	1	3	1
 286	Synthetic Rock	synthrock.gif	none	synthetic rock		0	0	0	0
 287	Grey Goose	greygoose.gif	item0	grey gosling	grey down vest	1	2	3	3
-288	Cookbookbat	bbat_fam.gif	item1,drop	mummified entombed cookbookbat	cookbookbat book	0	0	0	0
-289	Mini-Trainbot	tbot_on.gif	combat1,hp1,mp1	deactivated mini-Trainbot	overloaded Yule battery	0	0	0	0
-290	Hobo in Sheep's Clothing	hobosheep.gif	stat0,item0,drop	unoccupied sheep suit	half-height cigar	0	0	0	0
+288	Cookbookbat	bbat_fam.gif	item1,drop	mummified entombed cookbookbat	cookbookbat book	0	3	2	3
```

Comment:
well, colour me surprised! Is this the first weakness in an IoTM familiar?

Reply:
That is odd.

```
You enter Chef Bookicus against Ramathorn in an Ultimate Cage Match.
Chef Bookicus struggles for 13 rounds, but is eventually knocked out.

Chef Bookicus lost.
```
I don't see any special message. However:
```
<td>You enter Chef Bookicus against Ramathorn in an Ultimate Cage Match.<p><p>Chef Bookicus struggles for 13 rounds, but is eventually knocked out.<p>Chef Bookicus lost.</td>
```
There are two "p" tags before the first "result".

That would be the location for a "special weakness" message.

Here is a match with no special weakness:
```
You enter Chef Bookicus against Kid Ichorous in a Scavenger Hunt.
Chef Bookicus finds 39 items from the list.

Kid Ichorous finds 13 items.

Chef Bookicus is the winner, and gains 3 experience!

Congratulations! Only 4 more wins until you get a prize from the Arena Goodies Sack!
```
With the HTML:

```
<td>You enter Chef Bookicus against Kid Ichorous in a Scavenger Hunt.<p>Chef Bookicus finds 39 items from the list.<p>Kid Ichorous finds 13 items.<p>Chef Bookicus is the winner, and gains 3 experience!<p>Congratulations!  Only 4 more wins until you get a prize from the Arena Goodies Sack!</td>
```
So, I think we did the right thing - an extra line line before your familiar's result is where it report's your familiar's special weakness. There is such a line - but no message.